qa/testing.go

```diff
@@ -249,6 +249,49 @@ func (f ResourceFixture) setupClient(t *testing.T) (*common.DatabricksClient, se
 	}, nil
 }
 
+// Quoting from: https://github.com/hashicorp/terraform-plugin-sdk/blob/main/internal/configs/hcl2shim/values.go#L19
+// "UnknownVariableValue is a sentinel value that can be used
+// to denote that the value of a variable is unknown at this time.
+// RawConfig uses this information to build up data about
+// unknown keys.""
+const UnknownVariableValue = "74D93920-ED26-11E3-AC10-0800200C9A66"
+
+// Reference: HCL2ValueFromConfigValue from https://github.com/hashicorp/terraform-plugin-sdk/blob/main/internal/configs/hcl2shim/values.go#L199
```

Comment:
We're able to use this exactly as is?

Reply:
Yes if we change our tests to conform to the method. There were 2 options:
- Have a modified version that supports other types namely the one we use in our tests: `type data map[string]interface{}`. 
- Use the same version that is used in production (which the library calls) 

In option 1, we could make it more flexible by not calling panic and instead throw the error (since we are going to use this for unit tests only) but I think it would be good to have the same behaviour that is run in production.